app.py

```diff
@@ -383,7 +383,10 @@ def coins_inserted():
 def monitor_coins_and_button():
     """Monitors coins inserted and buttons pushed
     """
-    # time.sleep(0.5)
+    
+    # 200 ms sleep for CPU idle and other processes
+    time.sleep(0.1)
```

Comment:
I need to test this because i've already do this change (see history). I'm changing it back because of a runtime issue.